### DISCUSSION TITLE
Updating Eufy-Security-Client

### DISCUSSION THREAD

**Discussion by escapethewormhole** [fuatakgun/eufy_security#361] [Q&A] [answered]
Forgive my ignorance I am very new. I have a Eufy Cam that was added in bropat/eufy-security-client v2.0.0 and it appears this has not been updated in @fuatakgun 's Eufy_Security add-on. Is it possible to update just the security client version to the latest v2.1.0, and if so hopefully this won't break v0.9.1 of fuatakgun's add-on.

**Comment by thibaultmol** [comment]
(and then the mention of this issue https://github.com/bropat/eufy-security-client/issues/63 in the readme of this project can also be removed I assume?)

**Comment by cdnninja** [comment, marked as answer]
Are you able to test and report back? If it works I’ll update the repo as it looks to have some fixes in it.
